include/zephyr/bluetooth/audio/bap.h

```diff
@@ -50,6 +50,63 @@ extern "C" {
 /** An invalid Broadcast ID */
 #define BT_BAP_INVALID_BROADCAST_ID 0xFFFFFFFFU
 
+/**
```

Comment:
Is there a reason for selecting different sets of intervals for 7.5 ms and 10 ms SDU intervals? To keep things simple, I would suggest using a common factor of these. That is, use intervals 30, 60, or 90 ms.

Edit. The BAP specification in itself suggests to use connection intervals of 7.5 - 30 ms, or 50 - 70 ms, see https://www.bluetooth.com/specifications/specs/bap-1-0-2/. I would suggest to stick to that.